### PR TITLE
Added new language string  for Batch URL processor, changed language …

### PR DESCRIPTION
**Pull request by tonypartridge**
Pull Request for Issue # .
Improving consistency of Language strings.
### Summary of Changes
Added new language string to make it more clear. 
Renamed existing language string to keep consistency. 

Previously we see:
screenshot 2016-12-05 11 36 29

After changes we see:
screenshot 2016-12-05 11 39 47

### Testing Instructions
View com_redirects list of redirects 
### Documentation Changes Required

…string for New URL for consistency.

### CODE REVIEW

File: administrator/components/com_redirect/views/links/tmpl/default_addform.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	<div class="accordion-group">
 		<div class="accordion-heading">
 			<a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion2" href="#batch">
-				<?php echo JText::_('COM_REDIRECT_FIELD_NEW_URL_LABEL');?>
+				<?php echo JText::_('COM_REDIRECT_BATH_UPDATE_WITH_NEW_URL');?>
```
Comment (brianteeman) [inline administrator/components/com_redirect/views/links/tmpl/default_addform.php]: Batch not bath

File: administrator/language/en-GB/en-GB.com_redirect.ini [hunk 2]
```diff
@@ -29,7 +29,8 @@ COM_REDIRECT_FIELD_COMMENT_DESC="Sometimes it is helpful to describe the URLs fo
 COM_REDIRECT_FIELD_COMMENT_LABEL="Comment"
 COM_REDIRECT_FIELD_CREATED_DATE_LABEL="Created Date"
 COM_REDIRECT_FIELD_NEW_URL_DESC="Enter the URL to be redirected to."
-COM_REDIRECT_FIELD_NEW_URL_LABEL="Destination URL"
+COM_REDIRECT_BATH_UPDATE_WITH_NEW_URL="Batch update new URL(s)"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_redirect.ini]: Batch not bath please

**Comment by tonypartridge** [comment]
@brianteeman that's never a good start. Fixed.

**Comment by brianteeman** [comment]
Dont you need to update the string in the detail view as well?

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13090">issues.joomla.org/tracker/joomla-cms/13090</a>.</sub>

**Comment by tonypartridge** [comment]
Source URL now also changed to Expired URL

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on bb25c6520c8953c2878dc9ffec7175807898344a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13090">issues.joomla.org/tracker/joomla-cms/13090</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on bb25c6520c8953c2878dc9ffec7175807898344a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13090">issues.joomla.org/tracker/joomla-cms/13090</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13090">issues.joomla.org/tracker/joomla-cms/13090</a>.</sub>

**Pull request merged**